src/Compilers/Core/CodeAnalysisTest/MetadataReferences/MetadataReferenceTests.cs

```diff
@@ -56,18 +56,38 @@ public void CreateFrom_Errors()
         }
 #endif
 
-        [Fact]
-        public void CreateFromImage()
+        [Theory, CombinatorialData]
+        public void CreateFromImage_Assembly(bool immutableArray)
         {
-            var r = MetadataReference.CreateFromImage(ResourcesNet451.mscorlib);
+            var peImage = ResourcesNet451.mscorlib;
```

Comment:
>ResourcesNet451.mscorlib [](http://example.com/codeflow?start=26&length=24)

If we expect the operation to succeed with a module image, let's cover this case as well. For all API flavors.